### PR TITLE
Feature: Added setting to change the background color

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
Added color picker to the appearance page
image

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

File: src/Files.App/Helpers/AppThemeResourcesHelper.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Windows.UI;
+
+namespace Files.App.Helpers
+{
+	public class AppThemeResourcesHelper
```
Comment (d2dyno1) [inline src/Files.App/Helpers/AppThemeResourcesHelper.cs:7]: Use `sealed` keyword for classes as it improves performance 🤑

File: src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs [hunk 2]
```diff
@@ -127,6 +127,12 @@ public bool BundlesWidgetExpanded
 			set => Set(value);
 		}
 
+		public UInt32 AppThemeRootBackgroundColor
```
Comment (d2dyno1) [inline src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs:130]: You can add `///<inheritdoc/>` to those

**Review by d2dyno1 — CHANGES_REQUESTED** [review]

**Comment by d2dyno1** [comment]
From initial look-over, it looks good!

File: src/Files.App/Helpers/AppThemeResourcesHelper.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Helpers/AppThemeResourcesHelper.cs:7]: Resolved

File: src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs (hunk 2 above)
Comment (yaira2) [reply src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs:131]: Resolved

**Pull request merged**